fife/__init__.py

```diff
@@ -13,5 +13,4 @@
 PARTICULAR PURPOSE. See the GNU Affero General Public License for more details
 (a copy is included in the LICENSE file).
 """
-
-__version__ = "1.3.4"
+__version__ = open("VERSION.txt").read()
```

Comment:
Does open need a second "r" argument?

Reply:
Will this work when someone calls `import fife; fife.__version__"? I think it won't because VERSION.txt is in the parent directory.